docs/recipes/Cancellation.md

```diff
@@ -8,25 +8,24 @@ This can be done with the [`.takeUntil()`](http://reactivex.io/rxjs/class/es6/Ob
 import { ajax } from 'rxjs/observable/dom/ajax';
 
 const fetchUserEpic = action$ =>
-  action$.ofType(FETCH_USER)
-    .mergeMap(action =>
-      ajax.getJSON(`/api/users/${action.payload}`)
-        .map(response => fetchUserFulfilled(response))
-        .takeUntil(action$.ofType(FETCH_USER_CANCELLED))
-    );
+  action$.ofType(FETCH_USER).mergeMap(action =>
+    ajax
+      .getJSON(`/api/users/${action.payload}`)
+      .map(response => fetchUserFulfilled(response))
+      .takeUntil(action$.ofType(FETCH_USER_CANCELLED))
+  );
```

Comment:
this one (and the others like it) are yucky IMO 😢

I totally get why it would default to it, and in many traditional cases I would prolly like it. I just think in the case of epics they become more maintainable if the formatting is always consistent. e.g. if you had added a debounce above the mergeMap it wouldn't have put them all on one line